### PR TITLE
Add emobase support

### PR DESCRIPTION
**Pull request by rnbwdsh**
Thanks to @SEILSdataset for the help.
Feature is needed for some a course at JKU Linz / Austria.

### CODE REVIEW

File: opensmile/core/smile.py [hunk 1]
```diff
@@ -172,7 +172,7 @@ def _check_deltas_available(self):
             FeatureSet.eGeMAPS,
             FeatureSet.eGeMAPSv01a,
             FeatureSet.eGeMAPSv01b,
-            FeatureSet.eGeMAPSv02,
+            FeatureSet.eGeMAPSv02
```
Comment (frankenjoe) [inline opensmile/core/smile.py:175]:
```suggestion
            FeatureSet.eGeMAPSv02,
```

File: opensmile/core/define.py [hunk 2]
```diff
@@ -15,12 +15,14 @@ class FeatureSet(enum.Enum):
...
     * :attr:`eGeMAPSv01b`
     * :attr:`eGeMAPSv02`
+    * :attr:`emobase`
 
     For references, see the papers on:
 
     * `ComParE 2016`_
     * GeMAPS_
     * eGeMAPS_
+    * emobase
```
Comment (frankenjoe) [inline opensmile/core/define.py:25]: Do we have a paper we could reference?

Comment (hagenw) [reply opensmile/core/define.py:25]: If you look at https://audeering.github.io/opensmile/get-started.html there are even different `emobase` feature sets:
* `emobase`
* `emobase2`
* `emobase2010`

I guess this pull request adds `emobase`?

Comment (chausner-audeering) [reply opensmile/core/define.py:25]: I am not sure about the origins of emobase. @feyben should be able to comment.

Comment (rnbwdsh) [reply opensmile/core/define.py:25]: It's the main emobase file with minor adjustments to work with the python-opensmile.

According to the documentation of the main opensmile, there isn't really a paper source. The linked main documentation calls it the _The openSMILE/openEAR ‘emobase’ set_, but the original openEAR paper also doesn't mention emobase, just emodb which seems the dataset, the feature extractor has been used on.

Comment (frankenjoe) [reply opensmile/core/define.py:25]: If @chausner-audeering agrees, I suggest to continue without a reference for the moment. We can still add one later.

Comment (chausner-audeering) [reply opensmile/core/define.py:25]: Sure, I'll leave that up to you as maintainers of opensmile-python. :)

**Comment by frankenjoe** [comment]
@rnbwdsh thanks for your efforts. The `emobase` feature set is now available with version `2.0.2`.

**Pull request merged**